scrabble/README.md

```diff
@@ -4,21 +4,61 @@ Determine which of two Scrabble words is worth more.
 
 ## Background
 
-TODO
+In the game of [Scrabble](https://scrabble.hasbro.com/en-us/rules), players create words to score points, and the number of points is the sum of the point values of each letter in the word. 
 
-## Implementation Details
+For example, if we wanted to score the word `Code`, we would note that in general Scrabble rules, the `C` is worth `3` points, the `o` is worth `1` point, the `d` is worth `2` points, and the `e` is worth `1` point. Summing these, we get that `Code` is worth `3+1+2+1 = 7` points. 
 
-Complete the implementation of `scrabble.c` at right, such that it TODO
+## Implementation Details
 
-TODO
+Complete the implementation of `scrabble.c` at right, such that it determines the winner of a short scrabble-like game, where two players each enter their word, and the higher scoring player wins.
 
+* We've stored the point values of each letter of the alphabet in an integer array named `POINTS`. 
+  * For example, `A` or `a` is worth `1` point, `B` or `b` is worth `3` points, etc.
+* We've created a header for a helper function called `score()` that takes a string as input and returns an `int`. Whenever we would like to assign point values to a particular word, we can call this function. Note that this header is required for C to know that `score()` exists later in the program. 
```

Comment:
"prototype" instead of "header"

Reply:
And for anything that isn't something the student needs to actively do, I'd preface the bullet point with something like "Notice that" so that it's clear it's just something the student needs to understand, rather than actually write.